src/pages/index.js

```diff
@@ -9,6 +9,7 @@ export default () => {
     <Layout>
       <Helmet>
```

Comment:
We should be able to leverage the `htmlAttributes` attribute on the `<Helmet>` tag.

`htmlAttributes={{ lang: 'en' }}` should do the trick!